src/libopal.c

```diff
@@ -2392,6 +2392,42 @@ short print_asm_code(asm_cmd_e cmd_list[], FILE *dest_fp)
   /// Copy NASM footer file to dest_fp
 
   /// Create strings and their lengths
+  for (int i = 0; i < strs_len; i++)
+    {
+      fprintf (dest_fp, "msg%d: DB \"", i);
+      /// Read each string character
+      for (int j = 0; j < strlen(strs[i])+1; j++)
```

Comment:
You shouldn't have to go 1 past the strlen .. are you seeing issues without it ?

Reply:
No, I might have before my previous fixes, but you're right that it works with it now.